libs/designer/src/lib/ui/templates/connections/displayConnections.tsx

```diff
@@ -0,0 +1,34 @@
+import { Button } from '@fluentui/react-components';
+import type { Template } from '@microsoft/logic-apps-shared';
+
+export interface DisplayConnectionsProps {
+  connections: Record<string, Template.Connection>;
+}
+
+export const DisplayConnections = ({ connections }: DisplayConnectionsProps) => {
+  return (
+    <div>
+      <div>Template Connections</div>
+      {Object.keys(connections).map((connection, index) => {
+        return (
+          <div key={index}>
+            <b>
+              {index + 1}: {connection}{' '}
+              <Button
+                appearance="outline"
+                onClick={() => {
+                  console.log('TODO: show connection for this');
+                }}
+              >
+                CHOOSE
+              </Button>
+            </b>
+            <div>
```

Comment:
NIT: do you need to a div inside an empty div? I mean maybe move the inner div outside?